intro-HPC/ch_introduction.tex

```diff
@@ -323,15 +304,15 @@ \subsection{What programming languages can I use?}
 \subsection{What operating systems can I use?}
 \label{sec:what-operating-systems-can-i-use}
 
-All nodes in the \hpc cluster run under \operatingsystemSL, which is a specific
-version of \operatingsystemRHEL. This means that all programs (executables)
-should be compiled for \operatingsystemSL.
+All nodes in the \hpc cluster run under \operatingsystem, which is a specific
+version of \operatingsystembase. This means that all programs (executables)
+should be compiled for \operatingsystem.
 
-A user can connect from any desk computer in the \university network to the
-\hpc, regardless of the Operating System that he is using on his personal
+Users can connect from any desk computer in the \university network to the
```

Comment:
'desk' ? :-p 

connect to the \hpc from any computer that is connected to the \university network ?